app/representers/account_representer.rb

```diff
@@ -54,6 +55,8 @@ module AccountRepresenter
   end
 
   property :state
+  property :bought_cinstances_count
+  property :admin_name
```

Comment:
```suggestion
  property :admin_user_name
```
WDYT? because `admin` is not clear that it is refering to `admin user`, specially when there are account attributes called `admin_domain` and `admin_base_url`

Reply:
The problem with `admin_user_name` is that it can be confused with the admin's `user_name`. It could be `admin_display_name`?

Reply:
@josemigallas yes that's good enough I guess